### ISSUE TITLE
non-functional

### ISSUE THREAD

**Issue by daedaevibin** [mrrfv/open-android-backup#130]
## Device info

Manufacturer: Google
Android version: 14
Custom ROM/rooted? EvolutionX
Model: Pixel 6a

## System info

Available disk space: 301GB
Operating system: Windows 10 Pro

---

**Describe the issue below.**
All APT commands aren't found and don't work so I'll need a list of dependencies on what to install

Error Message:

Neither whiptail nor dialog are installed, can't continue. Please refer to the README for usage instructions.
Exiting.
Press Enter to continue...:

**Comment by mrrfv** [comment]
Have you read the installation and usage instructions in the README? You need to install WSL2 along with a distro like Ubuntu or Debian, then right click on `backup-windows.ps1` in the file explorer and run it. This script takes care of all dependencies and updates.

**Comment by daedaevibin** [comment]
> Have you read the installation and usage instructions in the README? You need to install WSL2 along with a distro like Ubuntu or Debian, then right click on `backup-windows.ps1` in the file explorer and run it. This script takes care of all dependencies and updates.

Yes, yes. I've done all of it. Nothing worked. It repeatedly says "sudo: apt: command not found" because you can't just put wsl in front of it anymore and have to actually enter the environment. Thing about that is you can't use adb from inside wsl either, because it won't read devices.

**Comment by daedaevibin** [comment]
Stale.
